ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/LgrDB.hs

```diff
@@ -175,7 +175,7 @@ openDB :: forall m blk.
           )
        => LgrDbArgs Identity m blk
        -- ^ Stateless initializaton arguments
-       -> Tracer m (TraceReplayEvent blk ())
+       -> Tracer m (TraceReplayEvent blk (Point blk, ()))
```

Comment:
Please change this argument type to `TraceReplayEvent blk (ReplayStart blk)`

Or to `ReplayGoal blk -> TracerReplayEvent blk`, if we go with my new suggestion.

Reply:
Done